addons/website/controllers/form.py

```diff
@@ -27,6 +28,12 @@ def website_form_empty(self, **kwargs):
         # This is a workaround to don't add language prefix to <form action="/website/form/" ...>
         return ""
 
+    @http.route('/website/form/check_type', type='jsonrpc', auth='user', methods=['POST'])
```

Comment:
Though we are using it for website purpose, can we move this controller in web -> controllers -> binary.py and rename it with /web/file/guess_mimetype? this way later on if someone want to guess mimetype from other backend modules then they can use this controller.